community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/DynamicProcessorAssigner.java

```diff
@@ -65,23 +65,20 @@ public void start( StageExecution execution )
     @Override
     public void check( StageExecution execution )
     {
-        int permits = availableProcessors - countActiveProcessors( execution );
         if ( execution.stillExecuting() )
         {
+            int permits = availableProcessors - countActiveProcessors( execution );
             if ( permits > 0 )
```

Comment:
Seems like it's not changed, but recalculated every time we call `DynamicProcessorAssigner.check( StageExecution execution )`